homeassistant/components/zwave_js/api.py

```diff
@@ -523,6 +525,47 @@ async def websocket_network_status(
     )
 
 
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/subscribe_controller_status",
+        vol.Required(DEVICE_ID): str,
+    }
+)
+@websocket_api.async_response
+@async_get_node
+async def websocket_subscribe_controller_status(
```

Comment:
Do we need this if we add a sensor?

Reply:
came to the same conclusion in the comment above. If we decide to not include this from the frontend, we should probably remove the node status from the device info card too. I can submit a PR for that if that's what we decide

Reply:
Sounds good.